### ISSUE TITLE
requirements_files list in `dante config`

### ISSUE THREAD

**Issue by cm-tm** [sbg/dante#1]
Unless I grossly misunderstood something, I reckon that there is a bug in `requirements_files` list when running `dante config`. For the following _setup.cfg_...

```
[dante]
allow_named_versions = true
named_version_patterns =
    0.*version

lock_file_path = requirements-dev.lock
requirements_files =
    requirements-dev.txt
lock_files =
    requirements-dev.lock
```

...I'd expect _requirements-dev.txt_ to be there instead of _requirements-dev.lock_.

```
{
    "dante": {
        "requirements_files": [
            "requirements-dev.lock"
        ],
        "lock_files": [
            "requirements-dev.lock"
        ],
    }
}
```

**Comment by QuicksilverMachine** [comment]
You're right, it's a bug when printing the config, where the lock files are printed instead of requirements files:
https://github.com/sbg/dante/blob/6abc37a175d2d89deb0387e116d240b9357537db/dante/config.py#L224-L225
Thanks for submitting. I'll fix it with the next release.

**Comment by QuicksilverMachine** [comment]
The issue is resolved with the release of 2.0.7.
